packages/trace-viewer/src/ui/networkFilters.tsx

```diff
@@ -41,16 +42,17 @@ export const NetworkFilters = ({ filterState, onFilterStateChange }: {
       />
 
       <div className='network-filters-resource-types'>
-        {resourceTypes.map(resourceType => (
-          <div
-            key={resourceType}
-            title={resourceType}
-            onClick={() => onFilterStateChange({ ...filterState, resourceType })}
-            className={`network-filters-resource-type ${filterState.resourceType === resourceType ? 'selected' : ''}`}
-          >
-            {resourceType}
-          </div>
-        ))}
+        <TabbedPane
```

Comment:
What about other places that use `TabbedPane`? Do they upgrade to the new functionality for free?